### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by jordigh**
Various proposed changes to the Readme, mostly minor

### CODE REVIEW

**Comment by artivis** [comment]
Thanks for the fixes.
Only one small remark left for review.

File: README.md [hunk 1]
```diff
@@ -50,14 +50,12 @@ Please find more information in the related [wiki page](https://github.com/artiv
...
 
-<p align="center">
-  <a href="#installation">Installation</a> •
-  <a href="#features">Features</a> •
-  <a href="#documentation">Documentation</a> •
-  <a href="#tutorials-and-application-demos">Tutorials</a> •
-  <a href="#publications">Publications</a> •
-  <a href="#contributing">Contributing</a>
-</p>
+* [Installation](#installation)
```
Comment (artivis) [inline README.md]: This changes the layout from a centered one-liner to a left-aligned list. I'd rather keep the original layout, can you revert this particular change ?

Comment (jordigh) [reply README.md]: Hm, github doesn't make it easy to accept part of a commit but not all of it, does it?

Forgive me, I'm more used to patches by email where this isn't a problem.

I'll rewrite the patch.

Comment (artivis) [reply README.md]: You can simply push another commit if this is simpler, I'll squash them at merge.

Comment (jordigh) [reply README.md]: I'll get around to it. I wish you could just do it without my intervention. Github is dumb.

**Pull request closed without merging**